src/config/routes.ts

```diff
@@ -47,4 +47,5 @@ export const AppRoutes = {
     index: '/transactions',
     history: '/transactions/history',
   },
+  relayingEducational: '/relaying-educational',
```

Comment:
Nit: looking back at this, `relaying-education` or simply `relaying` reads better but I doubt the user will focus on the URL.

Reply:
Just adding this for future reference: I thought about this being a route or a modal some more yesterday.

We could make it(/them) a modal(/modals) and blur the background like we do with other ones in the UI. (This would also avoid the sidebar issue I mentioned further up.)

Reply:
food for thought @TanyaEfremova @liliiaorlenko